mkosi/distributions/none.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: LGPL-2.1+
+
+import urllib.parse
+from collections.abc import Sequence
+
+from mkosi.architecture import Architecture
+from mkosi.distributions import Distribution, DistributionInstaller, PackageType
+from mkosi.log import die
+from mkosi.state import MkosiState
+
+
+class NoneInstaller(DistributionInstaller):
+    @classmethod
+    def filesystem(cls) -> str:
+        return "ext4"
+
+    @classmethod
+    def package_type(cls) -> PackageType:
+        return PackageType.rpm
```

Comment:
That seems an odd choice? Maybe we should have an associated package type `none`, too.

Reply:
Just an oversight. I also added "none", but without testing on my side so far.
#1933 also adds this, so should sort out itself.